core/web/resolver/p2p_test.go

```diff
@@ -0,0 +1,199 @@
+package resolver
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/smartcontractkit/chainlink/core/services/keystore"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/p2pkey"
+)
+
+func TestGetP2PKeys(t *testing.T) {
```

Comment:
Can you give these test names more specificity, like `TestResolver_GetP2PKeys` - I think that makes it easier to run all resolver tests doing things like `test ./... -run "TestResolver"`